### PR TITLE
Update bash install script

### PR DESCRIPTION
**Pull request by paulsheldrake**
The code that selects the version to install terminus from assumes that the most current version i.e 3.x versus 2.x is listed first.   If you are assuming the 3.x code is what is going to be downloaded and you have further 3.x specific features like plugins used then your script will fail.  

This designates a known fallback version if for some reason the 2.x release is listed first.

Closes #

## Summary

**[Doc Page Title](https://pantheon.io/docs/doc-title)** - <Enter a one sentence summation of the pertinent changes (including not-yet-completed work) provided by this PR.>

## Effect

The following changes are already committed:

*
*

## Remaining Work and Prerequisites

The following changes still need to be completed:

- [ ] List any outstanding work here

### Dependencies and Timing

- [ ] Other prerequisites that must be completed before merging this PR

**Release**:
- [ ] When ready
- [ ] After date: $DATE

--------------------------------------------------

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] For Heroes - add a props post to the [discussion board](https://discuss.pantheon.io/c/pantheon-platform/documentation/17).
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by kporras07** [comment]
Hi @joa-pan @paulsheldrake if we're hardcoding a version (as fallback in this case), I'd like to recommend a different approach for this:
- Always download a hardcoded version
- Run self:update command to get it updated


It would be like this:

```
mkdir ~/terminus && cd ~/terminus
curl -L https://github.com/pantheon-systems/terminus/releases/download/3.0.4/terminus.phar --output terminus
chmod +x terminus
./terminus self:update
sudo ln -s ~/terminus/terminus /usr/local/bin/terminus
```

What do you think?

**Comment by kporras07** [comment]
btw, yes. I agree on the need for this PR. Blindly relying on always getting the latest version and expecting it to be 3.x may be problematic

**Comment by paulsheldrake** [comment]
Hey Folks

Thanks for the feedback, I like it.   Have tweaked the PR

File: source/content/terminus/install.md [hunk 1]
```diff
@@ -42,11 +42,11 @@ The following commands will:
 - add a symlink to your local `bin` directory for the Terminus executable.
 
   ```bash{promptUser: user}
-  mkdir ~/terminus && cd ~/terminus
-  TERMINUS_RELEASE=$(curl --silent "https://api.github.com/repos/pantheon-systems/terminus/releases/latest" | perl -nle'print $& while m#"tag_name": "\K[^"]*#g')
-  curl -L https://github.com/pantheon-systems/terminus/releases/download/$TERMINUS_RELEASE/terminus.phar --output terminus
-  chmod +x terminus
-  sudo ln -s ~/terminus/terminus /usr/local/bin/terminus
+mkdir -p ~/terminus && cd ~/terminus
+curl -L https://github.com/pantheon-systems/terminus/releases/download/3.0.0/terminus.phar --output terminus
```
Comment (uberhacker) [inline source/content/terminus/install.md:46]: Shouldn't this be `3.0.5` instead of `3.0.0`?  Otherwise, `self:update` doesn't work, right?

Comment (joa-pan) [reply source/content/terminus/install.md:46]: Thank you so much @paulsheldrake, @uberhacker, and @kporras07! We really appreciate all your work on this 

Looking over what Kevin previously added, @uberhacker I think you are correct. @kporras07 can you please confirm?

Comment (greg-1-anderson) [reply source/content/terminus/install.md:46]: Yeah, let's make a 3.0.6 release f Terminus, and then use that here.

Comment (joa-pan) [reply source/content/terminus/install.md:46]: Great! Thanks a million 💯

Comment (joa-pan) [reply source/content/terminus/install.md]: Hey @greg-1-anderson could you please review - this has been updated to `3.0.6`

**Review by jspellman814 — APPROVED** [review]
LGTM

**Review by greg-1-anderson — APPROVED** [review]

**Review by joa-pan — APPROVED** [review]
LGTM

**Pull request merged**